### PR TITLE
Share AoI Census

### PR DESCRIPTION
**Pull request by jamesmcclain**
When a new scenario is created, the census of the Area of Interest (AoI) is automatically attached to the new scenario.  Since the AoI is the same for all scenarios in a project, this can be done.  Doing this reduces the load the Geoprocessing service.

Connects #897

**To Test**
   * Open up the developer tools so that you can see the network traffic generated by the application, and also watch the spark-jobserver logs.
   * Go to a project and create a new scenario.
   * There should be no polling for the AoI census with this patch applied, but without it there should be.
   * Caching works on the compare page, as well.

**Limitations**
When a new project is created, there will still be two separate queries: one for "current conditions" and one for the "new scenario".  Since the census of the AoI is not available at that point, this cannot be helped without delaying the creation of the "new scenario" scenario.

### CODE REVIEW

**Comment by caseypt** [comment]
Code looks good, but I'm going to do some more testing tomorrow. Nice work.

File: src/mmw/js/src/modeling/models.js [hunk 1]
```diff
@@ -494,6 +494,12 @@ var ScenarioModel = Backbone.Model.extend({
             ]
         });
 
+        // If the census of the Area of Interest is available, then
+        // attach it so that it does not need to be recomputed.
+        if (attrs.aoi_census) {
+            this.set('aoi_census', attrs.aoi_census);
+        }
```
Comment (caseypt) [inline src/mmw/js/src/modeling/models.js]: Does the above code block (488 - 495) not do the job of setting `aoi_census` as well as any other attributes passed in on initialization?

Comment (jamesmcclain) [inline src/mmw/js/src/modeling/models.js]: Hmm ... I think it probably does ...

**Comment by caseypt** [comment]
+1. Tested. Nice job.

**Comment by jamesmcclain** [comment]
Thanks, I will make the cleanups you suggested before merging.

**Pull request merged**